packages/form-control/src/FormControlMixin.ts

```diff
@@ -327,16 +327,23 @@ export function FormControlMixin<
         return false;
       }
 
-      const showError = this.#forceError || (this.#touched && !this.validity.valid && !this.#focused);
+      const showError = this.#forceError || (this.#touched && !this.validity.valid && !this.#focused),
+        states = this.internals.states,
+        // Casted to any, because the polyfill forces dashed-ident syntax https://github.com/calebdwilliams/element-internals-polyfill/issues/126
+        // TODO: Remove any cast when the issue is resolved
+        state: any = 'show-error';
```

Comment:
Remove inaccurate type annotation.

Reply:
There's a comment and a TODO explaining why it's necessary. To make it work I should remove the `import { IElementInternals } from "element-internals-polyfill";` from form-control types.